docs/api/browser-window.md

```diff
@@ -1471,6 +1471,16 @@ Adds a window as a tab on this window, after the tab for the window instance.
 Adds a vibrancy effect to the browser window. Passing `null` or an empty string
 will remove the vibrancy effect on the window.
 
+#### `win.setWindowControlsVisibility(visible)` _macOS_
+
+* `visible` Boolean
+
+Sets whether the window controls buttons should be visible.
+
+#### `win.isWindowControlsVisible()` _macOS_
```

Comment:
It's unfortunate that "is" refers to a single thing but "window controls" is plural. But I suppose `areWindowControlsVisible()` might also be strange. 🤔

Reply:
oh didn't see this until i got down here...i think either are technically fine but `areWindowControlsVisible()` is slightly more readable?

Reply:
Fixed in the latest commit